content/articles/two-factor-authentication-enforcement.markdown

```diff
@@ -0,0 +1,78 @@
+
+* TOC
+{:toc}
+
+---
+
+[Two-Factor authentication](/articles/two-factor-authentication/) provides an extra level of security to protect your user credentials. You can enforce two-factor authentication on account members by enabling the feature for the account.
+
+When you have multiple users managing an account, it's vitaly important to reduce at minimum the likelyhood of an user credential being compromised. This is why this feature is available to all accounts that has the ability to [manage members of an account](/articles/account-users/).
+
+## Enabling Two-Factor authentication enforcement {#enable}
+
+You can enable Two-Factor authentication enforcement from your account members page.
+
+You can only enable Two-Factor authentication enforcement an account if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature, then re-invite them, they will be asked to enable Two-Factor authentication after they sign up so they can join your account.
+
+<div class="section-steps" markdown="1">
+##### To enable Two-Factor authentication enforcement
+
+1.  Log in to DNSimple with your user credentials.
+1.  Go to your account settings by selecting the <label>Settings</label> or <label>Account Settings</label> link from the navigation.
+
+    ![Settings menu](/files/account-settings-access.jpg)
+
+1.  Select <label>Team</label> from the menu on the left.
+1.  Scroll down until you see the <label>Enforce two-factor authentication</label> section. When all members of your Team have Two-Factor authentication enabled you will be able to enable this feature right away.
+
+    ![Two-Factor authentication enforcement](/files/account-2fa-enforcement.png)
+
+1.  If there are users in your account that does not have Two-Factor authentication enabled you will see a list of users that are not compliant, you can [remove the users](/articles/account-users/#removing-members-from-an-account) from the account in the section above.
+</div>
+
+
+## Disabling Two-Factor authentication enforcement {#disable}
+
+<div class="section-steps" markdown="1">
+##### To disable Two-Factor authentication
+
+1.  Log in to DNSimple with your user credentials.
+1.  Go to your account settings by selecting the <label>Settings</label> or <label>Account Settings</label> link from the navigation.
+
+    ![Settings menu](/files/account-settings-access.jpg)
+
+1.  Select <label>Team</label> from the menu on the left.
+1.  Scroll down until you see the <label>Enforce two-factor authentication</label> section. There is a link to <label>Disable</label> the feature.
+
+    ![Disable Two-Factor authentication enforcement](/files/account-disable-2fa-enforcement.png)
+</div>
+
+
+## Disabling Two-Factor authentication while in an enforced team
+
+When you are part of an Two-Factor enforcing team, you can still [disable Two-Factor authentication](/articles/two-factor-authentication/#disable) on your user.
```

Comment:
> "When you are part of an Two-Factor enforcing team" seems more a special punishing squad

It reads quite badly to me. It seems you are referring to an action team.

Reply:
I disbanded the action squad.